js/src/analysis/getTitlePlaceholder.js

```diff
@@ -0,0 +1,26 @@
+/* global wpseoPostScraperL10n, wpseoTermScraperL10n */
+
+var isUndefined = require( 'lodash/isUndefined' );
+
+/**
+ * Returns the title placeholder for use in the title forms.
+ *
+ * @returns {string}
+ */
+function getTitlePlaceholder() {
```

Comment:
That was my initial take, but I am using this function outside the post and term scraper as well where I don't have the context about which l10n object to use.

Futhermore in the future I think we should go to a shared l10n object name and this abstraction makes that possible.